_posts/2021-05-01-sr2021-knockouts.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: news
+title: St Paul’s College storm to victory in Student Robotics 2021
+---
+
+Student Robotics 2021 Virtual Competition came to an exciting conslusion today,
+with the rookie team from St Paul’s College, Adelaide, storming to victory in a
+close match against Hills Road Sixth Form College third team.
+
+The Challenge: *Radars of the Lost Ark*
+----------------------------
+
+{% include figure.html
+           class="right"
+           src="/images/content/news/sr2021/arena.png"
+           caption="The final form of the virtual arena, as played in the fourth league session and knockouts." %}
+
+Our game this year, _Radars of the Lost Ark_, challenged teams to score points
+by claiming territories using radios. Teams had to navigate the arena scanning
+for tower transmissions and sending their own transmissions back to claim
+territories. However, only territories which were linked either to their
+starting zone or to territories already owned could be claimed. At the end of
+the match, teams were awarded points based on the territories they owned. Full
+details, including the prizes available this year, are available in the
+[rulebook][rules].
+
+The Leagues
+-----------
+
+Our competition event this year spread the league stage over four weekends
+throughout the year. After each league session the game gained additional
+modules that presented new and different challenges to the competitors. In
+addition to the formal league sessions, there were also some more casual
+friendlies sessions which allowed the teams to experiment with different
+strategies and test their code against other teams’ in a more relaxed
+environment.
+
+While this year’s competition presented a different challenge to the competitors
+that previous years’, the core challenge to create an autonomous robot remained
+the same.
+
+{% include figure.html
+           class="right"
+           link_to_image="true"
+           src="/images/content/news/sr2021/league-4-top-teams.png"
+           caption="Teams’ game points per league session." %}
+
+During the league sessions there were some teams which showed strong early
+performances. Initially the teams from Hills Road Sixth Form College, looked to
```

Comment:
```suggestion
performances. Initially the teams from Hills Road Sixth Form College looked to
```

Reply:
Probably worth putting their TLA in too, to follow the pattern below

```suggestion
performances. Initially the teams from Hills Road Sixth Form College (HRS1-4) looked to
```

Reply:
Comma removed. I'm not sure the TLAs adds much here?